commons/commons-util/src/test/kotlin/ebi/ac/uk/util/file/ExcelReaderTest.kt

```diff
@@ -30,6 +30,8 @@ class ExcelReaderTest(private val temporaryFolder: TemporaryFolder) {
                 row {
                     cell("Study")
                     cell("SECT-001")
+                    cell("")
```

Comment:
here we are complicating main test case to include corner case scenarios testing, I think instead, we should add an extra simple test case `when extra empty cells` and keep this one only for happy path.

Reply:
Done